### PR TITLE
Fix bootup bug

### LINKED ISSUE #2318: "meshery.yaml" not getting created when single command install is run
#### Description

The [single command install](https://docs.meshery.io/installation/quick-start#install-meshery) fails when it executes `mesheryctl system start`, as it doesn't find "meshery.yaml" created. Tried on various platforms.

Work around was to manually copy "meshery.yaml" [from here](https://github.com/layer5io/meshery/pull/174/files/37a12a5be2b290e26315f36c159da0af2d427968). 

```
Installing mesheryctl in /usr/local/bin.
mesheryctl installed.
permissions moved to root
Removing installation files and opening Meshery...Docker is not running.
Start Docker now [y/n]? Attempting to start Docker...
Prerequisite Docker started.
Docker-Compose is not installed
Attempting Docker-Compose installation...
Prerequisite Docker Compose is installed.
Error: failed to start Meshery
See https://docs.meshery.io/guides/mesheryctl/#meshery-lifecycle-management for usage details
: open /root/.meshery/meshery.yaml: no such file or directory
Usage:
  mesheryctl system start [flags]

Flags:
  -h, --help          help for start
      --reset         (optional) reset Meshery's configuration file to default settings.
      --skip-update   (optional) skip checking for new Meshery's container images.

Global Flags:
      --config string   config file (default location is: /root/.meshery/config.yaml)
```
I've confirmed that no "meshery.yaml" file gets created either (Is this expected?):

#### Expected Behavior

Should succesfuly create a default "meshery.yaml", and continue without issues.

#### Environment:
- Host OS: Ubuntu 20.04, Ubuntu 18.04, CentOS 7, WSL2
- Meshery Version: 0.5.0-beta-1
- Kubernetes Version: -
- Browser: -

<!-- Optional 
#### To Reproduce
(Side note: As of this release, although the [installation script](git.io/meshery) includes the `--silent` flag, the release is older, and doesn't include the flag, and spits out an error.) 

1. Run the [single command install](https://docs.meshery.io/installation/quick-start#install-meshery)

### PR DESCRIPTION
**Pull request by anirudhjain75**
**Description**
Fix the check that was inplace to create if meshery.yaml does not exist

This PR fixes #2318

### CODE REVIEW

File: mesheryctl/internal/cli/root/system/start.go [hunk 1]
```diff
@@ -86,7 +86,7 @@ func start() error {
 		}
 	}
 
-	if _, err := os.Stat(utils.MesheryFolder); os.IsNotExist(err) {
+	if _, err := os.Stat(utils.DockerComposeFile); os.IsNotExist(err) {
```
Comment (leecalcote) [inline mesheryctl/internal/cli/root/system/start.go:89]: What happens if the MesheryFolder itself doesn't exist?

Comment (anirudhjain75) [reply mesheryctl/internal/cli/root/system/start.go:89]: the above check is for that

Comment (anirudhjain75) [reply mesheryctl/internal/cli/root/system/start.go:89]: look into the condition above

**Review by kumarabd — APPROVED** [review]

File: mesheryctl/internal/cli/root/system/start.go (hunk 1 above)
Comment (leecalcote) [reply mesheryctl/internal/cli/root/system/start.go:89]: Thanks, @anirudhjain75. I missed the check for the presence of the directory above.

@navendu-pottekkat, a mental note to take into consideration for when all of the K8s manifests are retrieved.

Comment (navendu-pottekkat) [reply mesheryctl/internal/cli/root/system/start.go:89]: @leecalcote Noted.

**Pull request merged**